api/src/test/java/org/openmrs/module/openconceptlab/client/OclClientTest.java

```diff
@@ -132,4 +143,40 @@ public void parseDateFromPath_shouldParseDate() throws Exception {
 		
 		assertThat(date, is(calendar.getTime()));
 	}
+
+	@Test
+	public void getOclReleaseVersion_shouldGeOclReleaseVersion() throws IOException {
+		String subscriptionToken = "53fc72f0498a707a26e4d903c0f24c2db24d1e35";
+		String version = oclClient.getOclReleaseVersion(URL_WITH_VERSION, subscriptionToken);
+		assertThat(version, is("v1.0"));
+	}
+
+	@Test
+	public void getExportUrl_shouldConstructCorrectExportUrlWhenVersionIsPassedToUrl() throws URIException {
+		String collectionVersion = "v1.0";
+		GetMethod getMethod = oclClient.getExportUrl(URL_WITH_VERSION, collectionVersion);
+		assertThat(getMethod.getURI().toString(), is(URL_WITH_VERSION + "/export"));
+	}
+
+	@Test
+	public void getExportUrl_shouldConstructCorretExportUrlWhenVersionIsNotPassedToUrl() throws URIException {
+		String collectionVersion = "v2.0";
+		GetMethod getMethod = oclClient.getExportUrl(URL_1, collectionVersion);
+		assertThat(getMethod.getURI().toString(), is(URL_1 + "v2.0/export"));
+	}
+
+	@Test
+	public void removeLastUrlForwardSlashIfExist_shouldWorkAsExpected() {
+		String newUrl1 = oclClient.removeLastUrlForwardSlashIfExist(URL_1);
+		String newUrl2 = oclClient.removeLastUrlForwardSlashIfExist(URL_2);
+		String[] url1Chars = URL_1.split(OCL_API_BASE_URL);

```

Comment:
My intuition of the split function is for a list of tokens separated by a separator. OCL_API_BASE_URL does not naturally look like a separator. It is instead look like a prefix for the url. So using split in this case does not make the code obvious to understand.